README.md

```diff
@@ -7,7 +7,7 @@
 ![Docs](https://github.com/codait/pydax/workflows/Docs/badge.svg)
 ![Development Environment](https://github.com/codait/pydax/workflows/Development%20Environment/badge.svg)
 
-A simple Python API for downloading and loading datasets from IBM's [Data Asset Exchange](https://ibm.biz/data-exchange).
+PyDAX is a Python API that offers users a simple and quick way to download and load any dataset with an associated PyDAX schema in just a few lines of code.
```

Comment:
I don't think we should include technical details in the first sentence in introduction. How about this: (we should probably refine this further)

```suggestion
PyDAX is a Python API that enables easy, pragmatic, and elegant programmatical downloading and loading of datasets.
```

Reply:
That works for me